openstack/neutron/templates/daemonset-network_agents.yaml

```diff
@@ -0,0 +1,194 @@
+{{ if $.Values.agent.enable_daemonset }}
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: neutron-agents
+  labels:
+    system: openstack
+    type: backend
+    component: neutron
+spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        name: neutron-agents
+        system: openstack
+        type: backend
+        component: neutron
+    spec:
+      hostNetwork: true
```

Comment:
Could you also add more metadata on the pod? Like this here (https://github.com/sapcc/helm-charts/blob/master/openstack/neutron/templates/deployment-server.yaml#L27)

Reply:
Possibly replacing "api" with "agent"